src/fontra/core/classes.py

```diff
@@ -218,7 +218,7 @@ class Component:
 
 @dataclass(kw_only=True)
 class Anchor:
-    name: str
+    name: Optional[str] = None
```

Comment:
I have a small request:
```suggestion
    name: Optional[str]
```
(Please apply manually locally, it will affect the tests.)

It turns out, having a default value for "name", but not for x and y, makes x and y appear _first_ in the unstructured dict. By removing the default (which forces us to supply a name on construction, but it may be None), the "name" attribute will appear first again, which is how I like it.

You will most likely have to update the E json file as a result of this change.

Reply:
Agree. Name at the top is way better. Done.